quick-start/e2e/qa-data/plugins/contentWithOptions.sjs

```diff
@@ -1,73 +1,73 @@
-     /*
-     * Create Content Plugin
-     *
-     * @param id         - the identifier returned by the collector
-     * @param options    - an object containing options. Options are sent from Java
-     *
-     * @return - your content
-     */
-    function createContent(id, options) {
-      var doc = cts.doc(id);
-      var root = doc.root.toObject();
-    
-      var source;
-      var opts;
-    
-      // for xml we need to use xpath
-      if (root && xdmp.nodeKind(root) === 'element') {
-        source = root.xpath('/*:envelope/*:instance/node()');
-      }
-      // for json we need to return the instance
-      else if (root && root.envelope && root.envelope.instance) {
-        source = root.envelope.instance;
-        opts = options;
-      }
-      // for everything else
-      else {
-        source = doc;
-      }
-    
-      return extractInstanceProduct(source, opts);
-    }
-    
-    /**
-     * Creates an object instance from some source document.
-     * @param source  A document or node that contains
-     *   data for populating a Product
-     * @return An object with extracted data and
-     *   metadata about the instance.
-     */
-    function extractInstanceProduct(source, opts) {
-      // the original source documents
-      var attachments = source;
-    
-      var sku = xs.string(source.sku || source.SKU);
-      var title = xs.string(source.title);
-      var price = xs.decimal(source.price);
-      var value1 = xs.string(opts.hello);
-      var value2 = xs.string(opts.myNumber);
-      
-      // return the instance object
-      return {
-        '$attachments': attachments,
-        '$type': 'Product',
-        '$version': '0.0.1',
-        'sku': sku,
-        'title': title,
-        'price': price,
-        'opt1': value1,
```

Comment:
are we still using opt1 and opt2 for tests anywhere else?

Reply:
I do see opt1 in 1 traces test 'should verify the traces viewer page'
and 3 jobs test
'should verify the loaded data'
'should verify the harmonized data with sku as original property'
'should verify the harmonized data with SKU as original property'

Tests dont use opt2 anywhere.

Reply:
if I am not wrong, I think opts was missed out and adding them to the updated custom content plugin should be good enough. Will confirm with @ayuwono